client/components/random-button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default class Random extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    if (this.props.route.path === 'collection') {
+      return (
+        <>
+          <i className="fas fa-random randomButton" onClick={this.props.handleRandom}></i>
+        </>
```

Comment:
If you are only returning one react element, you can leave out the `<></>` fragment.